cmd/radio.go

```diff
@@ -185,18 +191,55 @@ type erasureConfig struct {
 	clnts  []bucketClient
 }
 
+func clientID(cfg remoteConfig) string {
+	hash := sha256.New()
+	jsonBytes, err := json.Marshal(cfg)
+	if err != nil {
+		return ""
```

Comment:
if clientID is empty what happens?

Reply:
defaulting to concatenated representation of bucket and endpoint if there is a failure in marshalling.